index.template.html

```diff
@@ -144,6 +144,14 @@ <h3><button class="closebutton" tabindex="-1"><i class="fa fa-times-circle"></i>
 <script src="//play.pokemonshowdown.com/data/moves.js?"></script>
 <script src="//play.pokemonshowdown.com/data/items.js?"></script>
 <script src="//play.pokemonshowdown.com/data/abilities.js?"></script>
+<script src="//play.pokemonshowdown.com/data/formats.js"></script>
+<script>
+	var FormatObj = {};
+	Formats.forEach(format => {
+		if (format.name) FormatObj[format.name] = format;
+	});
+	Formats = FormatObj;
+</script>
```

Comment:
why are you replicating the script here?

Reply:
wwoops